charts/postgres_config/templates/configmap.yaml

```diff
@@ -9,13 +9,13 @@ data:
     echo "SELECT pg_terminate_backend(pg_stat_activity.pid) FROM pg_stat_activity WHERE pg_stat_activity.datname = '$DB_NAME';" | psql -d postgres -w
```

Comment:
In this Helm Chart used anywhere? I think we used this in SPSH in the beginning but the name was not well defined, so we switched to use postgres_recreate_database. If this Helm Chart is not referenced anywhere, please remove